### PR TITLE
Update map and add knifes object layer

### PR DESCRIPTION
**Pull request by elderferreiras**
Updated the map to finally draw all the platforms from the original sketch. I had to make some adjustments so that jumping from one platform to another was a bit easier.

I also added a knifes object layer to properly place knifes in their designated places.

### CODE REVIEW

File: server/game/scene.js [hunk 1]
```diff
@@ -154,8 +154,10 @@ class GameScene extends Scene {
       this.movingPlatforms.add(new Escalator(this, this.getID(), escalator.x, escalator.y))
     })
 
-    // TODO(richard-to): Temporarily hardcoding position
-    this.knivesGroup.add(new Knife(this, this.getID(), 1600, 620))
+    // Add knifes
```
Comment (richard-to) [inline server/game/scene.js:157]: [nit] The plural of knife is knives

**Comment by richard-to** [comment]
Feel free to merge this in. Looks great.

File: server/game/scene.js (hunk 1 above)
Comment (elderferreiras) [reply server/game/scene.js:157]: lol that's embarrassing

**Comment by richard-to** [comment]
Can you also include your tiled map file in the resources?

**Comment by elderferreiras** [comment]
> Can you also include your tiled map file in the resources?

Not fully sure what you mean by this.

**Comment by elderferreiras** [comment]
oh wait now I know

**Comment by richard-to** [comment]
Do you have a `.tmx` file for your map?

**Comment by elderferreiras** [comment]
> Do you have a .tmx file for your map?

Nope. I just open the json file and make my changes in it.

**Comment by richard-to** [comment]
Ah, I see. That's interesting. I'll try that out.

**Comment by richard-to** [comment]
Nice that worked for me.

**Pull request merged**